### DISCUSSION TITLE
Including _geo filtering in route URL

### DISCUSSION THREAD

**Discussion by GabeMeister** [algolia/instantsearch#5146] [Q&A]
We are using Meilisearch with Algolia Instantsearch React hooks, and it covers a TON of use cases, which is awesome! Keep up the great work!

I did have a question regarding the `_geo` parameter. We are relying on this for searching on regions (for example, instead of refining by city=`Seattle` and region=`WA`, we're just utilizing the `_geo` parameter and doing 25 miles around the latitude and longitude of `Seattle` (which is `47.6062095` and `-122.3320708`). The reason we're doing this is because we want to include other items that were tagged with nearby cities (e.g. city=`Bellevue` and region=`WA`) all in the search results. 

However, I noticed that when we render a `<Configure />` component, it doesn't update the url as expected, like other `<RefinementList />` components do. I would have expected something like `_geo[lat]=47.6062095&_geo[lon]=-122.3320708` to show up as query params in the url, but it doesn't. This is problematic because when the user refreshes the page, they don't get the same results back. 

Is there something I'm missing here?

**Comment by Haroenv** [comment]
The default routing ignores configure, as in general it's used for settings. You can include them by passing stateMapping:

```
const stateMapping = {
  routeToState(routeState) { return routeState },
  stateToRoute(uiState) { return uiState },
};
```

of course you can still do other modifications, but this prevents the default from excluding configure
